### DISCUSSION TITLE
navbar problem

### DISCUSSION THREAD

**Discussion by Serhatpars** [jmjuanes/siimple#172] [Q&A]
Hello, my website has navbar active on desktop, but navbar does not go down on mobile, what could be the reason?

https://serhatpars.space

**Comment by jmjuanes** [comment]
Hello @Serhatpars. In your JavaScript code (`https://serhatpars.space/space.js`) please remove this part: 

```
export default {
    dialogs: {
        modal: {
            maxWidth: "500px",
        },
    },
    // ...other configuration
};
``` 

Remember that this is only to configure siimple and must be placed in the `siimple.config.js` file. See the CLI usage at https://siimple.xyz/docs/.

**Reply by Serhatpars** [reply]
I did what you said but for some reason I didn't get any results. Have I forgotten the truth that I know? 😅

**Reply by jmjuanes** [reply]
You are mixing browser-specific code with the configuration of siimple. In your `https://serhatpars.space/space.js` file please remove all the configuration of siimple and keep only this code: 

```js
const menu = document.querySelector("#navbar-menu");
const toggler = document.querySelector("#navbar-toggler");

toggler.addEventListener("click", () => {
    menu.classList.toggle("is-flex-mobile");
    menu.classList.toggle("is-hidden-mobile");
});
```

**Reply by Serhatpars** [reply]
As I don't understand, it didn't happen again.

**Reply by Serhatpars** [reply]
I couldn't figure out where I went wrong, but instead of calling it from the file, I wrote it directly into the script tag, the problem was solved. :D
